docker-compose.yml

```diff
@@ -218,6 +248,33 @@ services:
         condition: service_healthy
       meilisearch-setup:
         condition: service_completed_successfully
+  summary:
+    image: ghcr.io/aet-devops25/promptheus-summary:main
+    build:
+      context: .
+      dockerfile: summary/Dockerfile
+    ports:
+      - 8083:8083
+    restart: unless-stopped
+    environment:
+      spring.datasource.url: jdbc:postgresql://db:5432/${POSTGRES_DB:-postgres}
+      spring.datasource.username: ${POSTGRES_USER:-postgres}
+      spring.datasource.password: ${POSTGRES_PASSWORD:-CHANGE_ME}
+      management.otlp.tracing.endpoint: http://monitoring:4318/v1/traces
+      management.otlp.metrics.export.url: http://monitoring:4318/v1/metrics
+      management.otlp.logging.endpoint: http://monitoring:4318/v1/logs
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8083/actuator/health"]
+      interval: 10s
+      timeout: 3s
+      retries: 5
+      start_period: 1s
+      start_interval: 100ms
+    depends_on:
+      db:
+        condition: service_healthy
+      genai:
+        condition: service_started
```

Comment:
this is new. Something you are not telling me @WoH ?
🤔

Reply:
Thanks for catching that, I'll remove it